AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Itinerary/Details.cshtml

```diff
@@ -125,6 +127,20 @@
         </div>
     </div>
 }
+<form method="post" action="@Url.Action("Details", "Itinerary", new { area="Admin" })#requests" class="form-horizontal">
```

Comment:
There seem to be a couple of problems with this form. Firstly it's not sending an anti forgery token so we get blocked by MVC (bad request) on the POST. The best option to fix this is to use the form taghelpers which will actually render an anti-forgery token for us automatically.

```
<form asp-action="Details" asp-controller="Itinerary" asp-area="Admin">
    // You inputs etc here
</form>
```

Secondly, I'm not clear why the #requests string is in there currently?